public/locales/ru.json

```diff
@@ -21,46 +21,46 @@
       "year_2": "{{count}} years"
     },
     "sort": {
-      "alphabetical": "Алфавитный",
+      "alphabetical": "Алфавитный порядок",
       "expiry": "Срок действия"
     },
-    "transactionBanner": "You are here because of a transaction we sent you or a referral from another site. You have names below that need to be renewed soon!"
+    "transactionBanner": "Вы находитесь здесь, так как мы отправили Вам транзакцию, либо по рекомендации с другого сайта. Ниже указаны имена, которые нужно продлить."
   },
   "blockies": {
-    "noAddressFound": "No address found"
+    "noAddressFound": "Адрес не найден"
   },
   "c": {
     "Address": "Адрес",
     "Content": "Содержимое",
     "Controller": "Контроллер",
     "Expiration Date": "Срок действия",
-    "Resolver": "Резолвер",
-    "about": "О",
-    "addresses": "Addresses",
+    "Resolver": "DNS-клиент",
+    "about": "О нас",
+    "addresses": "Адреса",
     "cancel": "Отменить",
-    "cannotclaimDns": "You cannot claim names under .{{ name }} on this site.",
+    "cannotclaimDns": "Вы не можете занимать имена .{{ name }} на этом сайте.",
     "claim": "Заявить",
-    "connect": "Connect",
-    "disconnect": "Disconnect",
+    "connect": "Подключиться",
+    "disconnect": "Отключиться",
     "edit": "Редактировать",
     "email": "Электронный почта",
-    "expires": "Истекает",
+    "expires": "Срок истекает",
     "faq": "FAQ",
     "favourites": "Избранное",
     "learnmore": "Узнать больше",
-    "learnmoreenglishonly": "Learn More",
-    "migrate": "Мигрировать",
```

Comment:
The Russian word "migrate" means migration in the literal sense, that is, moving from one country to another.